crates/ruma-identity-service-api/src/association/check_3pid_validity/v2.rs

```diff
@@ -0,0 +1,52 @@
+//! [GET /_matrix/identity/v2/3pid/getValidated3pid](https://matrix.org/docs/spec/identity_service/r0.3.0#get-matrix-identity-v2-3pid-getvalidated3pid)
+
+use js_int::UInt;
+use ruma_api::ruma_api;
+
+ruma_api! {
+    metadata: {
+        description: "Determines if a given 3pid has been validated by a user.",
+        method: GET,
+        name: "check_3pid_validity",
+        path: "/_matrix/identity/v2/3pid/getValidated3pid/",
+        rate_limited: false,
+        authentication: AccessToken,
+    }
+
+    request: {
+        /// The Session ID enerated by the `requestToken` call.
```

Comment:
Typo, "generated" is missing the first letter.

Reply:
Also `requestToken` should probably be the module name we use for that endpoint, and be an intra-doc link like `[request_token](crate::whatever::request_token::v2)`. Same below.

Reply:
I'm not sure how to best do the intra-doc link, since it seems that there are 2 possible modules for `requestToken` calls:
- [ruma::api::identity_service::association::email::create_email_validation_session](https://docs.ruma.io/ruma/api/identity_service/association/email/create_email_validation_session/index.html)
- [ruma_identity_service_api::association::msisdn::create_msisdn_validation_session](https://docs.ruma.io/ruma_identity_service_api/association/msisdn/create_msisdn_validation_session/index.html)

Would it be a good idea to link to both? In other parts of the codebase, I saw it just being mentioned as a `requestToken` call, which is why I did it in the first place.

Reply:
Oh, I see. We can keep it as-is then (and possibly revisit later).